ricochet.pro

```diff
@@ -263,7 +263,7 @@ TRANSLATIONS += \
     translation/ricochet_zh_HK.ts
 
 isEmpty(QMAKE_LRELEASE) {
-    win32:QMAKE_LRELEASE = $$[QT_INSTALL_BINS]\lrelease.exe
+    contains(QMAKE_HOST.os,Windows):QMAKE_LRELEASE = $$[QT_INSTALL_BINS]/lrelease.exe
```

Comment:
@rburchell is there a nicer way to figure out the name of `lrelease`, or to figure out where there's a `.exe` suffix for the host (not target!) platform?